pamelalozano/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,31 @@
+
+package com.google.sps.servlets;
+
+
+import java.util.*;  
+import com.google.gson.Gson;
+import java.io.IOException;
+
+public class Comment {
+  
+  private String subject;
+  private String msg;
+
+  public Comment(String s, String m) {
+    this.subject=s;
+    this.msg=m;
+  }
+  
+  public void setSubject(String s){
+      this.subject=s;
+  }
+  public void setMessage(String m){
+      this.msg=m;
+  }
+  public String getSubject(){
+      return this.subject;
+  }
+  public String getMessage(){
+      return this.msg;
+  }
+};
```

Comment:
nit: add newline : )

Reply:
By newline you mean a new line before the " }; " ?

Reply:
I think he means the last line of the file should be a newline to get rid of the "No newline at end of file" warning.

Reply:
Yep this isn't too important though